.eslintrc

```diff
@@ -106,14 +116,22 @@
     "plugin:react-hooks/recommended",
     "plugin:no-unsanitized/DOM",
     "airbnb",
-    "prettier",
-    "prettier/react"
+    "prettier"
   ],
   "globals": {
     "global": false,
     "process": false
   },
   "overrides": [
+    {
+      "files": ["*.mdx"],
+      "extends": "plugin:mdx/recommended",
+      "rules": {
```

Comment:
I'll double check these rules as per your comment when I go through all the suggestions etc but for the record we've never had linting on mdx files and as I recall when I've added it these weren't possible to "fix" as there was nothing actually wrong. As I say, I will double check this though and you're right we could maybe do some investigation but there is a ticket created to go through all of our linting rules and strip it back as there's a lot of rules that we're putting in just because it's in airbnb's ruleset and I believe it would better to simplify this a fair bit and have more control for the team

Reply:
ah yes, I'd missed that we didn't have any linting on mdx before.

As I said, these seem to be failing for no good reason as they're on code that absolutely shouldn't fail these rules. Happy to have a new ticket to investigate so this PR doesn't get bogged down with it, but I don't want to just disable important accessibility-related checks and forget about it, even if we end up going with this as a temporary fix for now.